src/infrastructure/collection/scrapers/scraper_factory.py

```
@@ -37,7 +35,7 @@ def __init__(self, http_client=None) -> None:
             http_client = get_default_client()
         self._http_client = http_client
 
-    def create_for(self, setting: ScraperSetting) -> BaseScraper:
+    def create_for(self, setting: ScraperSetting, days_back: int = None) -> BaseScraper:
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)